buildpacks/jvm-function-invoker/opt/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+runtime_layer_jar_path="${1}"
+function_bundle_layer_dir="${2}"
+
+additional_java_args=()
+if [[ -n "${SF_FX_REMOTE_DEBUG}" ]]; then
+	java_version=$(java -version 2>&1 | grep -i version | awk '{gsub(/"/, "", $3); print $3}')
+
+	if [[ "${java_version}" == 1.8* ]]; then
+		additional_java_args+=("-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005")
+	else
+		additional_java_args+=("-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=*:5005")
+	fi
+fi
+
+java "${additional_java_args[@]}" \
```

Comment:
Should this use `exec`?

https://stackoverflow.com/questions/32255814/what-purpose-does-using-exec-in-docker-entrypoint-scripts-serve

Reply:
Will investigate, thanks for the pointer!

Reply:
Yes, it should 💯  use exec, thanks Ed! 👍🏻